AdminWebsite/AdminWebsite/ClientApp/src/app/shared/menus/venues-menu/venues-menu.component.html

```diff
@@ -0,0 +1,25 @@
+<div [formGroup]="form">
+  <label class="govuk-label" for="venue-list">Venues</label>
+  <ng-select
+    id="venue-list"
+    class="custom"
+    labelForId="venues"
+    [items]="venues"
+    bindLabel="name"
+    bindValue="id"
+    formControlName="selectedVenueIds"
+    [placeholder]="'Select venues'"
+    [multiple]="true"
+    [attr.aria-label]="'Venue list'"
+    [closeOnSelect]="false"
+    [clearSearchOnAdd]="true"
+    [selectOnTab]="true"
+    (add)="onSelect()"
+    (change)="onSelect()"
+  >
+    <ng-template ng-option-tmp let-item="item" let-item$="item$" let-index="index">
```

Comment:
similar to the previous question, why ng-template over ngfor. I would argue ngfor is an easier read